db/seeds/test_data.rb

```diff
@@ -919,3 +919,8 @@ def create_participant_with_eligibility(urn, name, participant_class, options =
     count: 10,
   ]
 end
+
+service = Importers::NPQContracts.new(
```

Comment:
To fix the tests after we rolled into cohort 2022, we amended the cohorts to 2022 for the test data (see https://github.com/DFE-Digital/early-careers-framework/blob/develop/db/seeds/test_data.rb#L369). So I think we'll be generating a lot of those fake ones now, with the old and new fixtures. Maybe we can rip the old creation out completely from `test_data` and only use the new way to import contracts, or create a new test csv for 2021 as well to create both?

Reply:
i've deleted all that code. it now imports 2021 npq contracts via csv import mechanism for which i've generated some fake data for